cmake/Kokkos_Version_Check.cmake

```diff
@@ -0,0 +1,11 @@
+function(check_minimum_required_kokkos kokkos_required_version)
```

Comment:
You should add a documentation in the header to describe what this function does.